EventFilter/HcalRawToDigi/plugins/PackerHelp.h

```diff
@@ -0,0 +1,466 @@
+#ifndef PACKER
+#define PACKER
+
+#include "DataFormats/HcalDetId/interface/HcalElectronicsId.h"
+#include "DataFormats/HcalDigi/interface/HcalDigiCollections.h"
+#include "DataFormats/FEDRawData/interface/FEDNumbering.h"
+#include "DataFormats/FEDRawData/interface/FEDRawData.h"
+#include "DataFormats/FEDRawData/interface/FEDHeader.h"
+#include "CondFormats/HcalObjects/interface/HcalElectronicsMap.h"
+#include "DataFormats/HcalDigi/interface/QIE10DataFrame.h"
+
+#include <iostream>
+#include "stdio.h"
+#include <vector>
+#include <map>
+
+using namespace std;
```

Comment:
even though it's a header in /plugins/, the ``` using namespace ``` is not allowed in headers in global namespace